lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,25 @@
 
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
 
-    codes = args.n
-    codes = codes.to_i
+  codes = args.n
+  codes = codes.to_i
 
-    codes.times do |x|
+  if args.u == nil
+    max_uses = 1
```

Comment:
We're already setting the default in the model. I don't think we will ever want to change it. But maybe we can only overwrite it below if it was handed as an arg?